behave_testrail_reporter/api.py

```diff
@@ -17,40 +17,40 @@ class APIClient:
     """
 
     def __init__(self, base_url):
-        self.user = os.environ.get(u'TESTRAIL_USER')
-        self.password = os.environ.get(u'TESTRAIL_KEY')
+        self.user = os.environ.get(u"TESTRAIL_USER")
+        self.password = os.environ.get(u"TESTRAIL_KEY")
 
-        if not base_url.endswith(u'/'):
-            base_url += u'/'
-        self.url = base_url + u'index.php?/api/v2/'
+        if not base_url.endswith(u"/"):
+            base_url += u"/"
+        self.url = base_url + u"index.php?/api/v2/"
         self.session = requests.Session()
         self.session.auth = (self.user, self.password)
-        self.session.headers.update({u'Content-Type': u'application/json'})
+        self.session.headers.update({u"Content-Type": u"application/json"})
 
         if not self.user or not self.password:
             raise ValueError(
-                u'Environment variables to authenticate on Testrail API are not defined!\n'
-                u'Please define those environment variables: \n'
-                u'TESTRAIL_USER and TESTRAIL_KEY'
+                u"Environment variables to authenticate on Testrail API are not defined!\n"
+                u"Please define those environment variables: \n"
+                u"TESTRAIL_USER and TESTRAIL_KEY"
             )
 
     def _build_endpoint_from_uri(self, uri):
-        return u'{base_url}{uri}'.format(base_url=self.url, uri=uri)
+        return u"{base_url}{uri}".format(base_url=self.url, uri=uri)
 
     def send_get(self, uri):
         endpoint = self._build_endpoint_from_uri(uri)
         response = self.session.get(endpoint)
         try:
             response.raise_for_status()
         except requests.HTTPError as e:
-            error_template = textwrap.dedent(u'''
+            error_template = textwrap.dedent(
+                u"""
```

Comment:
Need a `\` at the end of this line for `dedent()` to do anything useful.

That's an existing bug though

Reply:
Good point! 
Will fix this in a separate PR since this one does not include any behaviour code change. :)